File: fos-client/src/app/models/food-report.ts


    // $Classes/Enums/Interfaces(filter)[template][separator]
    // filter (optional): Matches the name or full name of the current item. * = match any, wrap in [] to match attributes or prefix with : to match interfaces or base classes.
    // template: The template to repeat for each matched item
    // separator (optional): A separator template that is placed between all templates e.g. $Properties[public $name: $Type][, ]

    // More info: http://frhagn.github.io/Typewriter/

    
     import { Comment } from './comment';

    export class FoodReport    {
           
        public FoodId: string = null;   
        public Name: string = null;   
        public Price: number = 0;   
        public Picture: string = null;   
        public Comments: Comment[] = [];   
        public TotalComment: string = null;   
        public Amount: number = 0;   
        public Total: number = 0;   
        public NumberOfUser: number = 0;   
        public UserIds: string[] = [];
    }
         

File: fos-api/FOS/FOS.Model/Domain/FoodReport.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FOS.Model.Domain
{
    public class FoodReport
    {
        public string FoodId { get; set; }
        public string Name { get; set; }
        public int Price { get; set; }
        public string Picture { get; set; }
        public List<Comment> Comments { get; set; }
        public string TotalComment { get; set; }
        public int Amount { get; set; }
        public int Total { get; set; }
        public int NumberOfUser { get; set; }
        public List<string> UserIds { get; set; }
    }
}
